team-telnyx/telnyx-client-examples
# Mute/unmute conference participant in UI

Mute and unmutes conference participants from browser app.

It's not the prettiest UI, but we have a card in TODO for a UI refresh.

## ✋ Manual testing

It's much easier to test this with two people, happy to test with you.

1. Run `call-center` app and log in
2. Call your call center phone number and answer. Add a test buddy's phone number to the conference (this will be very confusing to test if you try to do it yourself.) Verify you see the mute button next to participant's name/number
3. Mute and unmute your conference participant. Verify that audio mute/unmutes

## 🦊 Browser testing

### Desktop

### Mobile

## 📸 Screenshots

<img width="593" alt="Screen Shot 2020-10-02 at 3 10 52 PM" src="https://user-images.githubusercontent.com/4672952/94973927-da252a00-04c1-11eb-8fb6-4140736009e0.png">

<img width="573" alt="Screen Shot 2020-10-02 at 3 10 59 PM" src="https://user-images.githubusercontent.com/4672952/94973942-e9a47300-04c1-11eb-9233-a67e9288b23e.png">
